test/tests/test_specs/test_spec_merge/test_spec_merge.sh

```diff
@@ -19,19 +20,23 @@ pushd $(dirname $0)
 
 fission spec apply
```

Comment:
Can we use the --wait flag in this command instead of sleep later?

Reply:
Wait does infinite wait, if we are stuck in package build times would increase and may keep running for a longer duration. Probably need to have wait with some timeout.
I can try this as a separate PR where a fix would be applicable across all tests.